packages/react-charting/docs/TestPlans/AreaChart/ComponentTests.md

```diff
@@ -0,0 +1,60 @@
+**Area Chart – Component test plan**
+
+**Sub-components: Area, x-axis, y-axis and Legend**
+
+1. **Area: Area data, Area color (multi colors), Area label (show/hide)**
+1. **Legends: show/hide legends, highlight the corresponding Area on legend hover**
+1. **Callout: Default/custom callout**
+1. **Labels: x-Axis labels, y-Axis labels**
+
+|                            **Test steps**                             |                                                 **Validation**                                                  | **Tool used** |
+| :-------------------------------------------------------------------: | :-------------------------------------------------------------------------------------------------------------: | :-----------: |
+|                      Test 1: [Snapshot testing]                       |                                                                                                                 |               |
+|           - With only data prop, numerical data on x-axis.            |                                          Renders Area chart correctly                                           |      RTL      |
```

Comment:
Add a test case with string data on x axis. For this case the chart should not work.